docs/updating.md

```diff
@@ -37,8 +37,8 @@ Now edit the project to depend on the new Compose SNAPSHOT version:
 
 Edit [dependencies.kt](https://github.com/google/accompanist/blob/main/buildSrc/src/main/java/com/google/accompanist/buildsrc/dependencies.kt):
```

Comment:
@chrisbanes you are still referring to `dependencies.kt` which doesn't exist anymore (same after).

Reply:
Thanks: ae26aebd6f3cca8805b56ad5dd7e1b5d7241f17a